shell/platform/embedder/tests/embedder_test_context_gl.cc

```diff
@@ -110,7 +110,7 @@ uint32_t EmbedderTestContextGL::GetWindowFBOId() const {
 }
 
 void EmbedderTestContextGL::SetupCompositor() {
-  FML_CHECK(!compositor_) << "Already ssetup a compositor in this context.";
+  FML_CHECK(!compositor_) << "Already setup a compositor in this context.";
   FML_CHECK(gl_surface_)
```

Comment:
This is actually still wrong; the verb is "set up".

Reply:
Gah you’re right and I swear I’ve fixed that elsewhere to add to my shame :)